website/client/src/libs/highlightUsers.js

```diff
@@ -14,9 +14,14 @@ export function highlightUsers (text, userName, displayName) { // eslint-disable
       return fullMatched;
     }
 
+    let fixedStr = mentionStr;
+    if (mentionStr.includes('<em>')) {
+      fixedStr = mentionStr.replace('<em>', '_');
+      fixedStr = fixedStr.replace('</em>', '_');
+    }
```

Comment:
Well done, this should work fine!

However, you could also use a regular expression here to do the two things in a single call. Also since the replace match will simply not match if `<em>` is not present, you can drop the if check as well.

You should be able to replace the entire block by `let fixedStr = mentionStr.replace(/<\/?em>/g, '_');`. Notice that I'm using a _regular expression literal_ here, which is a javascript native way to easily represent regular expressions (You usually only use the `new RegEx` constructor if you're dynamically creating the regular expression, like is being done on line 7)

Reply:
This is great! I was thinking about using a RegEx to do it however I wasn't sure how to go about it so I tried to do it in this more simplified way :) I'll add it in!